iceoryx2-ffi/cxx/src/unique_port_id.cpp

```diff
@@ -43,6 +46,15 @@ UniquePublisherId::UniquePublisherId(iox2_unique_publisher_id_h handle)
     : m_handle { handle } {
 }
 
+auto UniquePublisherId::bytes() -> iox::optional<std::array<uint8_t, UNIQUE_PORT_ID_LENGTH>> {
+    if (m_handle != nullptr) {
+        std::array<uint8_t, UNIQUE_PORT_ID_LENGTH> unique_port_id {};
```

Comment:
I initially wanted the Rust layer to just provide a pointer to the ID bytes, however I couldn't work out how to resolve the ownership between the two layers, so I opted for a copy instead.
Do you see any better options?